sdk/servicebus/service-bus/test/internal/unit/receiverCommon.spec.ts

```diff
@@ -290,6 +293,68 @@ describe("shared receiver code", () => {
 
       assert.equal(numRetryCalls, 2 + 1);
     });
+
+    it("respects retry options", async () => {
+      const errorMessages: string[] = [];
+      let numRetryCalls = 0;
+
+      const fakeRetry = async <T>(): Promise<T> => {
+        ++numRetryCalls;
+
+        if (numRetryCalls < 4) {
```

Comment:
nit: could the constant 3 be factored into a variable everywhere so there is no magic numbers?